tests/storage/mpathconf_test.py

```diff
@@ -102,3 +102,64 @@ def test_read_bad_blacklist(fake_conf):
     fake_conf.write(BAD_CONF)
     wwids = mpathconf.read_blacklist()
     assert wwids == {"wwid1", "wwid2", "wwid5"}
+
+
+@pytest.fixture
+def fake_mpath_conf(tmpdir, monkeypatch):
+    fake_conf = tmpdir.join("multipath.conf")
+    monkeypatch.setattr(mpathconf, "CONF_FILE", str(fake_conf))
+    return fake_conf
+
+
+def test_mpath_current_tag_conf(fake_mpath_conf):
+    mpath_conf_current_tag = """\
+# VDSM REVISION 2.2
+"""
```

Comment:
Current target is must use the value from mpathconf.CURRENT_TAG since once we change it, the test will break, or we will have to change  the test each time we change the revision.

Reply:
I had it like that, and changed it following your comment here: https://github.com/oVirt/vdsm/pull/239#discussion_r901147725

Or did I misunderstand the comment?

Reply:
Using current tag does not mean we have to use complicated parameters, just include it in the tests that must use the current value.

Reply:
Got it.